src/main/java/com/hotel/room/getPlaces.java

```diff
@@ -0,0 +1,5 @@
+package com.hotel.room;
+
+public interface getPlaces {
+    public int getNumberOfFreePlaces();
```

Comment:
public access modifier is redundant, methods in interface are abstract and public by default

Reply:
Okey, fixed it.